tools/driver/autolink_extract_main.cpp

```diff
@@ -106,6 +107,31 @@ class AutolinkExtractInvocation {
   }
 };
 
+// Look inside the object file 'ObjectFile' and append any linker flags found in
+// its ".swift1_autolink_entries" section to 'LinkerFlags'. This will be used
+// for the recursive function 'extractLinkerFlags()'.
+static void
```

Comment:
Please use triple slashes for doc comments.

Please also drop the second sentence ("This will be used for the recursive function 'extractLinkerFlags()'.") as it just duplicates the information already present in the source code.